sdks/java/build-tools/src/main/resources/docker/README.md

```diff
@@ -0,0 +1,180 @@
+<!--
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+-->
+
+Docker Images
+========================
+
+This directory contains a set of scripts to build docker images of Apache Beam
+for different purposes:
+
+- **file**: Create a docker container from a .zip file with the Beam in a given
+  environment. It is useful to test/validate the release votes or an specific
+  version in a
+  sdks/java/build-tools/src/main/resources/docker/README.mddifferent version of
```

Comment:
hahaha oups that was when I as doing `git add` before the PR sorry, thanks for noticing.